server/svix-server/src/core/cryptography.rs

```diff
@@ -8,7 +8,8 @@ use chacha20poly1305::{Key, XChaCha20Poly1305, XNonce};
 use ed25519_compact::*;
 use rand::Rng;
 
-use crate::error::{Error, Result};
+use crate::error::Result;
+use crate::generic_err;
```

Comment:
I couldn't figure out how to scope the macros in `error.rs` in the actual `error` module, so we could call `error::generic!(...)` instead of `generic_err!(...)`. Suggestions there would be helpful.

Reply:
Oh right, macros are always crate-wide?

So maybe at least prefix it with err? So `err_generic` so it's at least namespaced like this, even if poorly.